message.go

```diff
@@ -14,5 +32,31 @@ type Message struct {
 }
 
 func (m Message) String() string {
-	return fmt.Sprintf("%s\t%s\t%s\t%s", m.Timestamp.Format(time.RFC3339), m.Level, m.Labels, m.Message)
+	var Color string
+	if term.IsTerminal(int(os.Stderr.Fd())) {
+		switch m.Level {
+		case "debug":
+			Color = DebugColor
+		case "warning":
+			Color = WarningColor
+		case "error":
+			Color = ErrorColor
+		case "info":
+			Color = InfoColor
+		default:
+			Color = DefaultColor
+		}
+	} else {
+		Color = ""
+		DefaultColor = ""
+	}
+	return fmt.Sprintf(
+		"%s%s\t%s\t%s\t%s%s",
+		Color,
+		m.Timestamp.Format(time.RFC3339),
+		m.Level,
+		m.Labels,
+		m.Message,
+		DefaultColor,
```

Comment:
Except that I've defined `AnsiReset` as a constant, and I need to override this end string as an empty string for the else condition of not being a terminal. This is why I already had is separately define as a `var` instead of a `const` before making this change.

Reply:
Oh, yeah; that's actually worse. Every time this code is called you're dynamically overriding the global variable and using it. And the names are bad for that use.

How about *local* variables `setColor` and `resetColor`, for example, to start and end the constructed string.